load.php

```diff
@@ -48,5 +48,10 @@
 		define( 'DB_NAME', 'test' );
 		define( 'EP_INDEX_PREFIX', 'tests_' );
 		define( 'WP_CACHE_KEY_SALT', 'codecept' );
+
+		// Support acceptance testing delayed execution. See AcceptanceTester::preLoadConfig.
+		if ( file_exists( Altis\ROOT_DIR . '/tests/load.php' ) ) {
+			include_once( Altis\ROOT_DIR . '/tests/load.php' );
```

Comment:
Per the other suggestion around file path:
```suggestion
		// Support acceptance testing delayed execution. See AcceptanceTester::bootstrapWith.
		if ( file_exists( Altis\ROOT_DIR . '/vendor/webdriver-test-load.php' ) ) {
			include_once( Altis\ROOT_DIR . '/vendor/webdriver-test-load.php' );
```